EMIP/1-1000/EMIP00816.xml

```diff
@@ -29,34 +29,226 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                   <collection>EMIP</collection>
                   <idno facs="EMIP/Codices/816/">Chelekot Silasie 75</idno>
                </msIdentifier>
+
```

Comment:
In the <titleStmt> section above, you should add your name as editor:

            <editor key="MKr"/>